### ISSUE TITLE
Stat not working at Ithaca

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#60]
In GitLab by @dmanubens on Oct 28, 2015, 18:10

@omula 

At Ihtaca, a002_20000101_fc0_7_CLEAN job started at: `Thu Oct 22 19:50` but strangely the last modification registerd for the .err file is much early in time: `Thu Oct 22 11:51`. 

```
dmanubens@login:/scratch/cfu/dmanubens/a002/LOG_a002$ stat -c %Z /scratch/cfu/dmanubens/a002/LOG_a002/a002_20000101_fc0_7_CLEAN.e151133 
1445507492
dmanubens@login:/scratch/cfu/dmanubens/a002/LOG_a002$ date -d @ 1445507492
Thu Oct 22 11:51:32 CEST 2015
dmanubens@login:/scratch/cfu/dmanubens/a002/LOG_a002$ grep "^+ job_start" /scratch/cfu/dmanubens/a002/LOG_a002/a002_20000101_fc0_7_CLEAN.e151133 
+ job_start_time=1445536221
dmanubens@login:/scratch/cfu/dmanubens/a002/LOG_a002$ date -d @ 1445536221
Thu Oct 22 19:50:21 CEST 2015
```

Happens the same with a002_20000101_fc0_8_CLEAN, a002_20000101_fc0_9_CLEAN and a002_20000101_fc0_10_CLEAN. Therefore the bar plot for statistics shows strange values for that jobs.

However in other jobs like a002_20000101_fc0_6_CLEAN the dates are ok:
```
dmanubens@login:/scratch/cfu/dmanubens/a002/LOG_a002$ stat -c %Z /scratch/cfu/dmanubens/a002/LOG_a002/a002_20000101_fc0_6_CLEAN.e151130
1445504280
dmanubens@login:/scratch/cfu/dmanubens/a002/LOG_a002$ date -d @ 1445504280
Thu Oct 22 10:58:00 CEST 2015
dmanubens@login:/scratch/cfu/dmanubens/a002/LOG_a002$ grep "^+ job_start" /scratch/cfu/dmanubens/a002/LOG_a002/a002_20000101_fc0_6_CLEAN.e151130
+ job_start_time=1445503684
dmanubens@login:/scratch/cfu/dmanubens/a002/LOG_a002$ date -d @ 1445503684
Thu Oct 22 10:48:04 CEST 2015
```

[a002_statistics_20151028_1634.pdf](https://earth.bsc.es/gitlab/es/autosubmit/uploads/2810b8b78e00a287882dad3c520c7cfc/a002_statistics_20151028_1634.pdf)

**Comment by kinow** [comment]
In GitLab by @omula on Oct 30, 2015, 14:47

Dates were wrong on the compute nodes. I have updated the ntp daemon of all of them.

**Comment by kinow** [comment]
In GitLab by @dmanubens on Nov 4, 2015, 11:01

Thanks @omula I will check if it is working fine with debug/autosubmit/a002 and a005

**Comment by kinow** [comment]
In GitLab by @dmanubens on Nov 5, 2015, 15:53

It's checked and fixed in 3.1.9
